.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
+before_install:
+  - gem install bundler
```

Comment:
👍 on using `tail`

Production is important, but out of the scope of this PR. That would be a PR for Ansible in `ofn-admin` repo.